app_dart/bin/server.dart

```diff
@@ -142,13 +142,15 @@ Future<void> main() async {
         final int trimIndex = min(queryIndex, anchorIndex);
         final String filePath = request.uri.path.substring(0, trimIndex);
 
-        const List<String> indexRedirects = <String>['/build.html'];
-        if (indexRedirects.contains(filePath)) {
+        const Map<String, String> redirects = <String, String>{
+          '/build.html': '/#/build',
+          '/repository': '/repository/index.html',
+          '/repository/': '/repository/index.html',
```

Comment:
Do we need these two options? Existing cocoon seems to only reference /repository.html. Or I missed somewhere.

Reply:
Yes. The Flutter app rewrites `/repository/index.html` to `/repository/#/`. On a reload, Cocoon see this as `/repository` and 404s. Ideally, we'd do something similar to below to just return the `index.html`, but it adds more complexity.